indico/modules/events/editing/client/js/components/Timeline/ReviewForm.jsx

```diff
@@ -58,31 +65,48 @@ export default function ReviewForm({block}) {
 
   const InputComponent = commentFormVisible ? FinalTextArea : FinalInput;
   const inputProps = commentFormVisible ? {autoFocus: true} : {};
+  const addComment = async (formData, form) => {
+    const rv = await dispatch(
+      createRevisionComment(eventId, contributionId, editableType, block.id, formData)
```

Comment:
considering how much data we are passing here just to build the URL I wonder if adding a `create_comment_url` in the marshmallow schema wouldn't be easier

Reply:
Yeah, maybe that is a good idea so I wouldn't have to pass additional props between components